src/pages/Login/Login.js

```diff
@@ -1,7 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Input from './Input';
+import Button from './Button';
+import Search from './Search';
+import ka from '../../assets/ka.png';
+const Login = () => {
+  const kakaoAuthUrl = process.env.REACT_APP_KAKAO_AUTH_URL;
+  const openKakaoLogin = () => {
+    window.location.href = kakaoAuthUrl;
+  };
+  const [userInfo, setUserInfo] = useState({
+    email: '',
+    pw: '',
+  });
```

Comment:
```suggestion
import ka from '../../assets/ka.png';

const Login = () => {
  const kakaoAuthUrl = process.env.REACT_APP_KAKAO_AUTH_URL;
  const openKakaoLogin = () => {
    window.location.href = kakaoAuthUrl;
  };
  
  const [userInfo, setUserInfo] = useState({
    email: '',
    pw: '',
  });
```
코드의 가독성을 높이는 것도 중요합니다!
간격을 띄워서 작성하는 연습도 해주세요~